src/main/java/com/checkmarx/jenkins/tools/CheckmarxInstaller.java

```diff
@@ -195,6 +237,44 @@ public Void call() throws IOException {
             return null;
         }
 
+        public static void copyURLToFile(URL source, String proxyStr, String proxyType, File destination, int connectionTimeoutMillis, int readTimeoutMillis) throws IOException {
```

Comment:
Im not sure I undewrstand the method name.. it seems like you are connection the proxy here no? so why not to name it connectProxy?

Reply:
I copied this function from FileUtils to be able to handle **also** connection with proxy.
the function opens a connection from the URL (with or without proxy) and reads a file from there.